### PR TITLE
P0 - Avatar & Badge components

### PR DESCRIPTION
**Pull request by Naveenchandar**
**Avatar Component**

image

**Badge Component**

image

### CODE REVIEW

**Review by ranitasahaa1230 — APPROVED** [review]
Looks good to me!

File: badge.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+.badge_number > span {
+    display: inline-block;
+    padding-top: 20%;
+    padding-bottom: 20%;
+}
+
+.badge_container > button {
+    background-color: #3f51b5;
+    color: white;
```
Comment (Aditi0505) [inline badge.css]: Please use hex code, for consistency.

File: avatar.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    width: 2.5rem;
+    height: 2.5rem;
+    font-size: 1rem;
+}
+
+.avatar_text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: red;
```
Comment (Aditi0505) [inline avatar.css:44]: Suggestion: Please use hex code for consistency.

**Review by Aditi0505 — APPROVED** [review]

File: badge.css (hunk 1 above)
Comment (Naveenchandar) [reply badge.css]: Done

File: avatar.css (hunk 2 above)
Comment (Naveenchandar) [reply avatar.css:44]: Done

**Pull request merged**